DentOS_Framework/DentOsTestbedLib/gen/model/dent/traffic/traffic.yaml

```diff
@@ -56,3 +59,11 @@
         type: string_list
         desc: |
           Traffic Generator Ports represented in "ip:slot:port"
+      - name: src
```

Comment:
Maybe it will be better to name it src_ip and dst_ip? In the future, we may need new variables like source/destination TCP/UDP ports and it will become confusing